sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -4,8 +4,11 @@
 
 #### Features Added
 * Added throughput control support for `gateway mode`. See [PR 36687](https://github.com/Azure/azure-sdk-for-java/pull/36687)
+* Added a preview API to `ChangeFeedProcessorBuilder` to process an additional context for handling all versions and deletes changes. - See [PR 36715](https://github.com/Azure/azure-sdk-for-java/pull/36715) 
 
 #### Breaking Changes
+* If `ChangeFeedProcessorBuilder` is configured with both overloads of `handleAllVersionsAndDeletesChanges` (with and without context handling)
```

Comment:
Why is this a breaking change?

Reply:
`handleAllVersionsAndDeletesChanges` now throws an `IllegalArgumentException`.

Reply:
but it is because the customer has configured both consumer? in that case, it is more like a config error instead of breaking change

Reply:
Makes sense - fixed in next iteration.